posthog/utils.py

```diff
@@ -186,6 +187,12 @@ def render_template(template_name: str, request: HttpRequest, context=None) -> H
         context["git_rev"] = get_git_commit()
         context["git_branch"] = get_git_branch()
 
+    if settings.DEBUG:
+        context["js_posthog_host"] = "http://localhost:8000"
```

Comment:
Minor thing, should we consider the edge case of users running this on different ports or host?

Reply:
I'll try out whether there's a way to use e.g. `window.location` here :)